tests/kernel/queue/src/test_queue_contexts.c

```diff
@@ -29,6 +29,16 @@ static K_THREAD_STACK_DEFINE(tstack1, STACK_SIZE);
 static struct k_thread tdata1;
 static struct k_sem end_sema;
 
+ZTEST_BMEM static int ret = TC_PASS;
+
+void k_sys_fatal_error_handler(unsigned int reason, const z_arch_esf_t *pEsf)
+{
+	if (reason != K_ERR_KERNEL_OOPS) {
```

Comment:
I think in the fullness of time it would be best to add optional fatal error handling directly into Ztest so that various test cases don't all have to do it themselves.